.werft/jobs/build/deploy-to-preview-environment.ts

```diff
@@ -221,34 +216,14 @@ async function deployToDevWithInstaller(werft: Werft, jobConfig: JobConfig, depl
 
     // clean environment state
     try {
-        if (deploymentConfig.cleanSlateDeployment && !withVM) {
-            werft.log(installerSlices.CLEAN_ENV_STATE, "Clean the preview environment slate...");
-            // re-create namespace
-            await cleanStateEnv(deploymentKubeconfig, metaEnv());
-
-        } else {
-            werft.log(installerSlices.CLEAN_ENV_STATE, "Clean the preview environment slate...");
-            createNamespace(namespace, deploymentKubeconfig, metaEnv({ slice: installerSlices.CLEAN_ENV_STATE }));
-        }
+
+        werft.log(installerSlices.CLEAN_ENV_STATE, "Clean the preview environment slate...");
+        createNamespace(namespace, deploymentKubeconfig, metaEnv({ slice: installerSlices.CLEAN_ENV_STATE }));
```

Comment:
Looks like a bug that already existed before your clean-up, but I'm not sure if we want to `always` create a new namespace in the "Clean slate deployment" phase.

That's even more relevant for harvester preview because we use the `default` namespace there, and there is absolutely no need to create such namespace 🤔 

Should we just delete this whole try-catch block?